test/test_helper.rb

```diff
@@ -2,10 +2,12 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 require 'mocha/mini_test'
+require 'capybara/rails'
 
 Dir["#{Rails.root}/test/support/**/*"].each { |file| require file }
 
 class ActiveSupport::TestCase
   # Add more helper methods to be used by all tests here...
   include FactoryGirl::Syntax::Methods
+  include Capybara::DSL
```

Comment:
@tgxworld  `Capybara::DSL` is required in this test, Shall I include only in this test ?